misc/SW_WA_2021_2024/guilderton_to_geraldton/swals/README.md

```diff
@@ -0,0 +1,390 @@
+# Tsunami model runs for Midwest region of WA, FY2023-2024
+--------------------------------------------------
+
+This folder contains code to run the tsunami models, including:
+* Scenarios like historical events [(see here)](../../Greater_Perth/sources/like_historic/)
+* Random PTHA18 scenarios [(see here)](../../Greater_Perth/sources/hazard/)
+
+The tsunami model includes the area from Seabird to Geraldton in high resolution.
+It is similar to earlier work on the Greater Perth area, but here we were able to use some new elevation datasets, refined breakwalls, measured river depths and include some previously excluded areas.
+
+In addition the code has been substantially updated to be easier to use and modify, the model setup is more efficient, and the scripts are setup to use the new SapphireRapids nodes on Gadi.
+
+An outline of the steps to run on NCI is provided below, followed by a summary of the key codes.
+See comments within the code for further documentation.
+You might also want to review [../GENERAL_GUIDANCE_ON_MODEL_SETUP.md](../GENERAL_GUIDANCE_ON_MODEL_SETUP.md).
+
+# Outline of steps to run on NCI
+
+``` bash
+
+#
+# COMPILE THE MODEL
+#
+
+# Get the right modules
+module clear
+source modules_SWALS_ifort_2023_B.sh
+# NOTE: I have seen one model that segfaults with these compilers (NSW model, Sumatra with time-varying source).
```

Comment:
Pretty sure this comment applies to the older modules using `ifx`-- not the one here.